cmd2.py

```diff
@@ -700,6 +708,10 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, use_ipython=False
 
     # -----  Methods related to presenting output to the user -----
 
+    def _finalize_app_parameters(self):
+        self.commentGrammars.ignore(pyparsing.quotedString).setParseAction(lambda x: '')
+        self.shortcuts = sorted(self.shortcuts.items(), reverse=True)
```

Comment:
In the future, we might want to add a comment documenting that self.shortcuts starts out as a dictionary, but here we are turning it into a sorted list of tuples, sorted in reverse order by shortcut.